big_tests/dynamic_domains.spec

```diff
@@ -10,41 +10,10 @@
 {include, "tests"}.
 
 {suites, "tests", acc_e2e_SUITE}.
-{suites, "tests", domain_isolation_SUITE}.
```

Comment:
Would it be possible to have alphabetical order of suites here, like for `default.spec`?